oluwajuwon/Fast-Food-Fast
# Chore implement feedback #160876105

### What does this pull request do?
This pull request edits the position of the user interface header and makes it fixed

### Description of the Task to be completed?
Users should be able to see the header containing the navigation as they scroll down the pages

### How should this be manually tested?
Users can test it by:
 - Visit https://oluwajuwon.github.io/Fast-Food-Fast/UI/index.html
 - Scroll down through the pages to check if the header position changes


### What are the relevant pivotal tracker stories?
The relevant pivotal story is:

Header should be fixed - https://www.pivotaltracker.com/story/show/160876105

### Screenshots
![8](https://user-images.githubusercontent.com/29441898/46258720-4cc03100-c4c7-11e8-8eee-7d6f7b79ec1c.PNG)
![9](https://user-images.githubusercontent.com/29441898/46258721-4cc03100-c4c7-11e8-96ac-41dfd12817c1.PNG)
